File: interlayer/src/main/java/com/street/interlayer/model/Card.java
package com.street.interlayer.model;

import com.fasterxml.jackson.annotation.JsonIgnore;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

@JsonIgnoreProperties(ignoreUnknown = true)
@Data
public class Card {

    private String id;
    @JsonProperty("owner")
    private Long ownerId;
    private Account account;
    private String number;
    private String exp_date;
    private String name;

}

File: micro/src/main/kotlin/com/street/micro/entity/Card.kt
package com.street.micro.entity

import com.fasterxml.jackson.databind.annotation.JsonSerialize
import com.fasterxml.jackson.databind.ser.std.ToStringSerializer
import groovy.transform.ToString
import org.bson.types.ObjectId
import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document

@Document
data class Card(
    @Id
    @JsonSerialize(using = ToStringSerializer::class)
    val id: ObjectId = ObjectId.get(),
    val owner: Long,
    val number: String,
    val exp_date: String,
    val name: String
)
